mscore/editstyle.ui

```diff
@@ -367,6 +367,35 @@
                </property>
               </spacer>
              </item>
+             <item row="2" column="0">
+              <widget class="QLabel" name="label_92">
+               <property name="text">
+                <string>Text font:</string>
+               </property>
+              </widget>
+             </item>
+             <item row="2" column="1">
```

Comment:
It's a clever implementation, but I think the UI needs work.  Right now it's just a combo box, meaning it isn't meaningful if - as may often be the case - there are a mix of fonts used.  Also, after using the combo box to change the fonts, then saving the score and reloading, it will still show FreeSerif, will it not?  Really, the control should probably be hidden behind an explicit Change button to make clear the combo box isn't showing you anything in particular relevant to the score.  Also, it might be nice if the code to initialize the combo box somehow *did* present something relevant, like maybe the current value of staff text, or maybe title.

Reply:
> It's a clever implementation, but I think the UI needs work. Right now it's just a combo box, meaning it isn't meaningful if - as may often be the case - there are a mix of fonts used.

My original idea was to create something like this (see image). In the first box you select the font that you want to change (you can only select one of the fonts that are in use in the current style), in the second box you select the font that you want to change it to and then you click the button. This way you can replace a font with another which is probably better than the original request.

>  Also, it might be nice if the code to initialize the combo box somehow did present something relevant, like maybe the current value of staff text, or maybe title.

I could initialize the combo box to the most used current font.

![changeText](https://user-images.githubusercontent.com/21060365/76687702-1f216600-662f-11ea-9ca5-4e11d94777e2.png)